pkg/apis/meta/v1alpha1/image.go

```diff
@@ -6,6 +6,10 @@ import (
 	"net/url"
 	"strings"
 
+	// "github.com/opencontainers/go-digest" requires us to load the algorithms
+	// that we want to use into the binary. (it calls algorithm.Available)
+	_ "crypto/sha256"
```

Comment:
Got this from @stealthybox 's branch https://github.com/stealthybox/ignite/blob/fix-oci-ref/pkg/apis/meta/v1alpha1/image.go#L9-L11 . Without this, the test run fails with "unsupported digest algorithm" error.